blosc/schunk.c

```diff
@@ -112,14 +117,20 @@ blosc2_schunk *blosc2_new_schunk(blosc2_cparams cparams, blosc2_dparams dparams,
 
 /* Create an empty super-chunk */
 blosc2_schunk *blosc2_empty_schunk(blosc2_cparams cparams, blosc2_dparams dparams, int nchunks,
-                                 blosc2_frame* frame) {
-  if (frame != NULL) {
-    fprintf(stderr, "Creating empty frames is not allowwed yet\n");
-    return NULL;
+                                   blosc2_storage* storage) {
+  if (storage == NULL) {
+    storage = (blosc2_storage*)malloc(sizeof(blosc2_storage));
+    memcpy(storage, &BLOSC2_STORAGE_DEFAULTS, sizeof(blosc2_storage));
   }
 
   blosc2_schunk* schunk = calloc(1, sizeof(blosc2_schunk));
 
+  if (storage->sparse == false) {
```

Comment:
Create an empty persistent schunk is not allowed yet.

Reply:
Addressed in 8db4718.